src/main/java/com/dev/cinema/dto/CinemaHallResponseDto.java

```diff
@@ -1,10 +1,16 @@
 package com.dev.cinema.dto;
 
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
 import lombok.Data;
 
 @Data
 public class CinemaHallResponseDto {
     private Long id;
+    @Min(value = 10, message = "All halls' capacities are greater than 10")
     private int capacity;
```

Comment:
I think you wanted to add these annotations in CinemaHall**Request**Dto instead?